src/main/java/seedu/address/logic/Messages.java

```diff
@@ -47,5 +47,15 @@ public static String format(Person person) {
         person.getTags().forEach(builder::append);
         return builder.toString();
     }
+    /**
+     * Formats the {@code person} for display to the user when marking paid
+     */
+    public static String markPaidFormat(Person person) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(person.getName())
+                .append("; MonthsPaid: ");
+        person.getMonthsPaid().forEach(month -> builder.append(month).append(" "));
```

Comment:
Any reason why `builder::append` wasn't used? Referring to implementation found in `format`

Reply:
Using builder:append wont append the " ". itll result in the UI printing all the months paid together. Ie. 2024-022023-022018-03

Reply:
Suggest you conform to `builder::append` and change toString of `monthPaid` instead, similar to implementation for Tag